dashboard/src/actions/overviewActions.js

```diff
@@ -26,9 +26,12 @@ export const getDatasets = () => async (dispatch, getState) => {
     params.append("mine", "true");
 
     const endpoints = getState().apiEndpoint.endpoints;
-    const response = await API.get(endpoints?.api?.datasets_list, {
-      params: params,
-    });
+    const response = await API.get(
+      uriTemplate(endpoints, "datasets_list", {}),
+      {
+        params,
+      }
```

Comment:
What does this code do?

I think `params` is already an object, so I don't really know what wrapping it in braces does.  The code here used to create an object/mapping with it as the value and `params:` as the key...but we're no longer doing that -- is that intentional?  (Analogous code elsewhere specifies the key, but in those cases the value is an ad-hoc object, not a `URLSearchParams`....)

Reply:
`{ params }` is implicitly `{params: params}`. And `params` is a field in the Axios options object, to generate query parameters, which is the 2nd parameter in `get` and `delete` and the third in `put` and `post` (after the payload body). This gives us `http://10.1.170.200:8080/api/v1/datasets?metadata=global.dashboard.saved&metadata=global.dashboard.seen&metadata=dataset.access&metadata=dataset.owner&metadata=dataset.uploaded&metadata=server.deletion&metadata=user.dashboard.favorite&mine=true`

Reply:
> `{ params }` is implicitly `{params: params}`.

Ah.  Thank you, Javascript....

> And params is a field in the Axios options object, to generate query parameters

Understood.  It took me awhile to decide that it was better than doing the string manipulation ourselves in the simple cases (obviously, the `URLSearchParams` approach is better here).